packages/viz/examples/styles/helpers/colorCategoriesStyle.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GeoJSON Layer with CARTO Tiles Example</title>
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" type="text/css" href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css">
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body>
+    <section id="map"></section>
+
+    <!-- Map libraries-->
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.0-alpha.6/dist.min.js"></script>
+
+    <!-- CARTO Deck.gl's Tile Layer -->
+    <script src="/dist/viz.umd.js"></script>
+
+    <script>
+      async function initialize() {
+        carto.setDefaultCredentials({ username: 'public' });
+
+        const deckInstance = new window.deck.DeckGL({
+          container: 'map',
+          mapStyle: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+          initialViewState: {
+            longitude: -3.700385991742541,
+            latitude: 40.41532659629741,
+            zoom: 12,
+          },
+          controller: true,
+        });
+
+        const roomTypeStyle = carto.colorCategoriesStyle('room_type', {
+          categories: ['Entire home/apt', 'Private room', 'Shared room'],
+
+          // categoryColors accepts an array of HEX color strings,
+          // or a string pointing to a CARTOColors palette
+          categoryColors: ['#88ccee', '#cc6677', '#ddcc77'],
```

Comment:
Should we just rename `categoryColors` to `colors` parameter? This array is just in the context of colorCategoriesStyle, so...

Reply:
It's a good idea now that we have `nullColor` and `othersColor`. I'm gonna rename `binColors` and `categoryColors` to just colors.